src/Drupal/Commands/config/ConfigImportCommands.php

```diff
@@ -177,14 +177,10 @@ public function __construct(ConfigManagerInterface $configManager, StorageInterf
     /**
      * Import config from a config directory.
      *
-     * @command config:import
```

Comment:
Can you see if the validate() hook in this file still runs? Even if it does, I think it should move to the new file along with its command definition.

Reply:
It is still working:

```
$ ./vendor/bin/drush config:import --partial
 [error]  Enable the config module in order to use the --partial option.
```

I'll move it to the new file anyway.